dist/index.js

```diff
@@ -2878,7 +2881,22 @@ function main() {
             cliMicrosoft365Path = yield (0, io_1.which)("m365", true);
             core.info("✅ Completed installing CLI for Microsoft 365.");
             core.info("ℹ️ Logging in to the tenant...");
-            yield executeCLIMicrosoft365Command(`login --authType password --userName ${username} --password ${password}`);
+            if (username && password) {
+                yield executeCLIMicrosoft365Command(`login --authType password --userName ${username} --password ${password}`);
```

Comment:
As we're adding tenantId and appId, why not expose them to all login modes rather than just for the certificate?

Reply:
Right, thank you for pointing that out!